config/routes.config.js

```diff
@@ -35,6 +35,10 @@ function getRoutes() {
         name: 'add_medication',
         path: { en: '/medication/add', fr: '/medication/add' },
       },
+      {
+        name: 'edit_medication',
+        path: { en: '/medication/:id/edit', fr: '/medication/:id/edit' },
```

Comment:
Can we have a proper french path here or some sort of path that reminds us this is not final text?

Reply:
done